### PR TITLE
Updating app versionName automatically

### LINKED ISSUE #1264: Get app version number from build.gradle and set automatically
**Actual Behaviour**
Currently devs need to update the app version manually from strings.xml

**Expected Behaviour**
To auto retrieve app version number and display in drawer menu
https://stackoverflow.com/questions/4616095/how-to-get-the-build-version-number-of-your-android-application

### PR DESCRIPTION
**Pull request by yatri1609**
Fixes #1264

**Changes**: 
I have done some changes in the build.gradle so that each time the app VersionName gets updated the string which is made in the gradle will update itself and indirectly update the app Version shown in the drawer menu. 
My reference link is : https://medium.com/@manas/manage-your-android-app-s-versioncode-versionname-with-gradle-7f9c5dcf09bf
Hence now the version in the drawer menu is getting updated automatically.

**Screenshot/s for the changes**: 

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[app_version_automated.zip](https://github.com/fossasia/pslab-android/files/2216086/app_version_automated.zip)

### CODE REVIEW

**Comment by yatri1609** [comment]
@cweitat Please have a look!

**Comment by cweitat** [comment]
@yatri1609 can add a word `Version` in front of the numbers? also travis failed

**Comment by cweitat** [comment]
Version Icon also please change

**Comment by cweitat** [comment]
@yatri1609 travis

**Comment by yatri1609** [comment]
@cweitat Ya I checked travis but I'm not getting why is it getting failed as the app is not crashing anywhere and even the build is successful in android studio. I'm still checking where it is getting failed.

**Comment by yatri1609** [comment]
@CloudyPadmal Can you help me with this as I'm not getting why is the travis build getting failed?

File: app/build.gradle [hunk 1]
```diff
@@ -14,71 +14,76 @@ android {
...
         }
     }
-    lintOptions {
-        abortOnError false
-    }
-}
 
-dependencies {
-    implementation fileTree(include: ['*.jar'], dir: 'libs')
+    dependencies {
```
Comment (CloudyPadmal) [inline app/build.gradle]: Hello @yatri1609 ! Please undo all these indents.

**Comment by CloudyPadmal** [comment]
@yatri1609 travis build log says `error: resource string/app_version (aka org.fossasia.pslab:string/app_version) not found.` Please remove references to that it will pass.

**Comment by yatri1609** [comment]
@CloudyPadmal Please can you once again have a look

**Comment by yatri1609** [comment]
@cweitat and @CloudyPadmal Please review it now

**Pull request merged**